src/app/login/login.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Router } from '@angular/router';
+import { UserService } from '../services/user.service';
+import { User } from '../user';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  @Input() user: User;
+  resp: string;
+  @Output() submitted = new EventEmitter<boolean>();
+  constructor(private userService: UserService, private router: Router) {
+    let submitBtn = document.getElementById('submit');
+    submitBtn.addEventListener('click', (e: Event) => this.submitRegistration());
+  }
+
+  submitRegistration(){
+    let newUser = new User();
+    newUser.email = document.getElementById('emailR').value;
```

Comment:
i have a strong feeling that grabbing the value like this will not work. can we try to use angular bindings instead?